src/main/java/mate/academy/springboot/datajpa/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springboot.datajpa.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
```

Comment:
same question here. Note: you can restrict the access by using @Setter with access level parameter